pack/tic/def_encounter.json

```diff
@@ -0,0 +1,598 @@
+		"back_illustrator": "Jorge Ramos Márquez",
+		"back_name": "Devil Reef",
+		"clues": 0,
+		"code": "07173",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 11,
+		"faction_code": "mythos",
+		"illustrator": "Emilio Rodriguez",
+		"name": "Black Reef",
+		"pack_code": "def",
+		"position": 173,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - After you reveal Black Reef: Increase its flood level. Put 2 random set-aside Tidal Tunnels into play, each adjacent to Black Reef (but not to one another). Then, put 1 random set-aside Unfathomable Depths into play adjacent to both of those Tidal Tunnels.",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+		"back_illustrator": "Patrick McEvoy",
+		"back_name": "Tidal Tunnel",
+		"clues": 1,
+		"code": "07174a",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 12,
+		"faction_code": "neutral",
+		"illustrator": "Álvaro Calvo Escudero",
+		"name": "Bootlegger's Hideaway",
+		"pack_code": "def",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After Bootlegger's Hideaway is revealed: Randomly choose 1 of the set-aside facedown keys and place it on Bootlegger's Hideaway without looking at it.\n[action] If you control the white key: Draw 3 cards. (Group limit once per game.)",
+		"traits": "Cave.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+		"back_illustrator": "Patrick McEvoy",
+		"back_name": "Tidal Tunnel",
+		"clues": 1,
+		"code": "07174b",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 13,
+		"faction_code": "neutral",
+		"illustrator": "Álvaro Calvo Escudero",
+		"name": "Bootlegger's Hideaway",
+		"pack_code": "def",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After Bootlegger's Hideaway is revealed: Randomly choose 1 of the set-aside facedown keys and place it on Bootlegger's Hideaway without looking at it.\n[action] If you control the black key: Gain 5 resources. (Group limit once per game.)",
+		"traits": "Cave.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```

Comment:
Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).